firmware/boron-ins-fsm/test/consoleFunctionTests.cpp

```diff
@@ -437,4 +441,71 @@ SCENARIO("Force_Reset", "[force reset]") {
             }
         }
     }
-}
\ No newline at end of file
+}
+
+SCENARIO("Reset_Stillness_Timer_For_Alerting_Session", "[reset stillness timer for alerting session]") {
+    GIVEN("A session that has already had a Duration Alert") {
+        unsigned long initial_state3_stillness_timer_value = 30005;
+        state3_stillness_timer = initial_state3_stillness_timer_value;
+        hasDurationAlertBeenSent = true;
+        hasStillnessAlertBeenSent = false;
+
+        WHEN("the function is called with any string") {
+            int returnVal = reset_stillness_timer_for_alerting_session("any string");
+
+            THEN("The function should return the difference between old stillness timer value and now") {
+                // Testing a one-second range to prevent test flakiness
+                unsigned long expectedValue = millis() - initial_state3_stillness_timer_value;
+                REQUIRE((returnVal >= expectedValue - 1000 && returnVal <= expectedValue));
+            }
+
+            THEN("The new stillness timer value is the current value of millis()") {
+                // Testing a one-second range to prevent test flakiness
+                int currentTimeInMillis = (int)millis();
+                REQUIRE((state3_stillness_timer >= currentTimeInMillis - 1000 && state3_stillness_timer <= currentTimeInMillis));
+            }
+        }
+    }
+
+    GIVEN("A session that has already had a Stillness Alert") {
+        unsigned long initial_state3_stillness_timer_value = 30005;
+        state3_stillness_timer = initial_state3_stillness_timer_value;
+        hasDurationAlertBeenSent = false;
+        hasStillnessAlertBeenSent = true;
+
+        WHEN("the function is called with any string") {
+            int returnVal = reset_stillness_timer_for_alerting_session("any string");
+
+            THEN("The function should return the difference between old stillness timer value and now") {
+                // Testing a one-second range to prevent test flakiness
+                unsigned long expectedValue = millis() - initial_state3_stillness_timer_value;
+                REQUIRE((returnVal >= expectedValue - 1000 && returnVal <= expectedValue));
+            }
+
+            THEN("The new stillness timer value is the current value of millis()") {
+                // Testing a one-second range to prevent test flakiness
+                int currentTimeInMillis = (int)millis();
+                REQUIRE((state3_stillness_timer >= currentTimeInMillis - 1000 && state3_stillness_timer <= currentTimeInMillis));
+            }
+        }
+    }
```

Comment:
All three tests look good - out of curiosity is it possible for both the hasDurationAlertBeenSent & hasStillnessAlertBeenSent to be true?